identity-apps-core/apps/authentication-portal/src/main/webapp/org_discovery.jsp

```diff
@@ -45,6 +45,8 @@
                errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "error.retry");
            } else if (errorMessage.equalsIgnoreCase("Can't identify organization")) {
                 errorMessage = AuthenticationEndpointUtil.i18n(resourceBundle, "invalid.organization.discovery.input");
+           } else if (errorMessage.equalsIgnoreCase("Organization discovery type is invalid or not enabled")) {
```

Comment:
Cannot we send an i18n string from the backend as the error message for this?

Reply:
In current backend implementation, a detailed error message (ex: "Can't identify organization") is sent and from the front end, the i18n key is distinguished using this error message. Therefore, to be consistent with that approach, the detailed error message is sent in this case as well.

Reply:
> Therefore, to be consistent with that approach,

That approach is wrong though :D. Refer to https://github.com/wso2/product-is/issues/18740. This is causing problems with encoding param inputs and may result in broken error messages.

It'd be better to update the BE implementation if possible.

cc: @brionmario

Reply:
Ack

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6805/commits/ad85637313d822491b988f19b6964d3ac86a1ecf and the following BE PR.
- https://github.com/wso2-extensions/identity-auth-organization-login/pull/46

Reply:
Ack. thanks